src/components/modules/AverageTimeReports/AverageTimeReports.tsx

```diff
@@ -0,0 +1,92 @@
+/*eslint-disable*/
+import { format } from 'date-fns';
+import React, { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import { type AirQuality } from '@/components/modules/AverageTimeReports/types';
+import data from '@/mocks/avg-data.json';
+
+type FormValues = {
+  dateFrom: string;
+  dateTo: string;
+};
+
+const DATE_FORMAT = 'YYYY-MM-DD';
+
+function AverageTimeReports() {
```

Comment:
Here's just my personal preference to put `export default` before the function declaration. It's only to keep project consistent.